vm/x86/jit_x86.cpp

```diff
@@ -1116,6 +1117,14 @@ Compiler::visitREBASE(cell_t addr, cell_t iv_size, cell_t data_size)
   return true;
 }
 
+bool
+Compiler::visitBREAK()
+{
+  __ call(&debug_break_);

```

Comment:
As discussed this should not be emitted unless the debugger is attached. It will definitely have a performance cost.